src/libxtp/CMakeLists.txt

```
@@ -4,7 +4,11 @@ add_custom_target(gitversion-xtp COMMAND ${CMAKE_COMMAND} -DTOP_SOURCE_DIR="${CM
 set_property(DIRECTORY APPEND PROPERTY ADDITIONAL_MAKE_CLEAN_FILES gitversion.h)
 
 file(GLOB_RECURSE VOTCA_SOURCES *.cc *.cpp)
-file(GLOB_RECURSE NOT_VOTCA_SOURCES version_nb.cc )
+file(GLOB_RECURSE NOT_VOTCA_SOURCES version_nb.cc)
```

```suggestion
file(GLOB_RECURSE NOT_VOTCA_SOURCES version_nb.cc eigencuda.cc)
```
And remove the `if` block below.

---

thanks!